VL.ImGui.Skia/src/SkiaWidget.cs

```diff
@@ -60,13 +63,14 @@ protected override void Draw(Context context, in ImDrawListPtr drawList, in Syst
         public RectangleF? Bounds => !_disposed ? Layer?.Bounds : default;
 
         // What is this for?
-        //SKMatrix? trans;
+        // it is somehow necessary for the Stride/Skia inputHandleing to work ... is a strange workaround
```

Comment:
Because the CallerInfo in Notify has the wrong transformation ... The CallerInfo in Render has the correct ... just set a breakpoint in Notify ... then you can see it

Reply:
Interesting - so this might be a bug in the Stride-Skia bridge then?

Reply:
Yes, probably that

Reply:
Will have a look..